api/server/router/network/backend.go

```diff
@@ -0,0 +1,22 @@
+package network
+
+import (
+	// network config needs to be pull out to here
+	"github.com/docker/docker/daemon/network"
+
+	"github.com/docker/libnetwork"
+)
+
+// Backend is all the methods that need to be implemented to provide
```

Comment:
Yea, that should probably be a real godoc doc too.
`// Backend represents blah blah blah`
I am kind of surprised none of the checkers caught it. Is this package not linted?